app/assets/javascripts/psique/menu_lvl2.js.coffee

```diff
@@ -0,0 +1,5 @@
+$('.menu-lvl2--current .icon').click (event) ->
+  icon = event.target
```

Comment:
Si no me equivoco, esto se puede sustituir por $(this), o mejor aun, en coffee, @:

```
$(@).toggleClass('fa-caret-down')
$(@).toggleClass('fa-caret-up')
```